ArduPlane/altitude.cpp

```diff
@@ -79,25 +79,15 @@ void Plane::setup_glide_slope(void)
             reset_offset_altitude();
         }
         break;
-
     case Mode::Number::AUTO:
-
-        //climb without doing glide slope if option is enabled
+        //climb without doing slope if option is enabled
         if (!above_location_current(next_WP_loc) && plane.flight_option_enabled(FlightOptions::IMMEDIATE_CLIMB_IN_AUTO)) {
             reset_offset_altitude();
             break;
         }
 
-        // we only do glide slide handling in AUTO when above 20m or
-        // when descending. The 20 meter threshold is arbitrary, and
-        // is basically to prevent situations where we try to slowly
-        // gain height at low altitudes, potentially hitting
-        // obstacles.
-        if (adjusted_relative_altitude_cm() > 2000 || above_location_current(next_WP_loc)) {
-            set_offset_altitude_location(prev_WP_loc, next_WP_loc);
-        } else {
-            reset_offset_altitude();
-        }
+        set_offset_altitude_location(prev_WP_loc, next_WP_loc);
```

Comment:
good to keep comment (update)

Reply:
I find it helpful to say why you call a function whose actions arent clearly described in its name, as well as complete comment on the function definition itself....not a blocker, just might  help someone in the future understand whats going on